packages/golden-layout/src/js/LayoutManager.js

```diff
@@ -222,6 +232,12 @@ lm.utils.copy(lm.LayoutManager.prototype, {
    * @returns {Function}
    */
   getComponent: function (name) {
+    if (this._components[name] === undefined) {
```

Comment:
I think this would conflict with the fallback functionality

Reply:
I'm going to roll this and the other changes into my other cleanup changes (separating plugins into their own package).

Reply:
Cleaned this up

Reply:
Can you push the update?